lib/src/client.dart

```diff
@@ -7,6 +7,7 @@ class NextCloudClient {
   NextCloudClient(
     Uri host,
     NextCloudHttpClient httpClient,
+    String username,
```

Comment:
This will not work, since at this point we usually have a login name. But the WebDav API needs the real username.